cli/src/commands/shell/index.ts

```diff
@@ -1,17 +1,10 @@
 import { Command } from '@oclif/core';
+import { run } from '@xata.io/shell';
 
 export default class Shell extends Command {
-  static description = 'Open a shell to the current database';
-
-  static examples = [];
-
-  static flags = {};
-
-  static args = [];
+  static description = 'Open a shell to the current database and branch';
 
   async run(): Promise<void> {
-    // const { args, flags } = await this.parse(Shell);
-
-    this.error('To be done');
+    await run({});
```

Comment:
Probably makes sense to make ``options`` optional

Reply:
Yep, yep. I'll do a PR for that.